lti_emailer/canvas_api_client.py

```diff
@@ -21,6 +21,21 @@
 SDK_CONTEXT = SessionInactivityExpirationRC(**settings.CANVAS_SDK_SETTINGS)
 
 
+def get_section(canvas_course_id, section_id):
```

Comment:
Eh, still not a fan of this approach, but I've rewritten it to use that approach, to maintain consistency in the app if nothing else.